game/scripts/npc/abilities/alchemist_unstable_concoction.txt

```diff
@@ -62,7 +62,7 @@
       "06"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "max_damage"                                      "150 220 290 360 660 960"
+        "max_damage"                                      "150 220 290 360 800 1200"
```

Comment:
If you change dmg or stun duration in `alchemist_unstable_concoction.txt` then you need to change them in `alchemist_unstable_concoction_throw.txt` too.